src/src/rx_main.cpp

```diff
@@ -878,4 +966,148 @@ void loop()
 #ifdef PLATFORM_STM32
     STM32_RX_HandleUARTin();
 #endif
+
+    // If the eeprom is indicating that we're not bound
+    // and we're not already in binding mode, enter binding
+    if (!config.GetIsBound() && !InBindingMode)
+    {
+        Serial.println("RX has not been bound, enter binding mode...");
+        EnterBindingMode();
+    }
+
+    // If the power on counter is >=3, enter binding and clear counter
+    if (config.GetPowerOnCounter() >= 3)
+    {
+        config.SetPowerOnCounter(0);
+        config.Commit();
+        
+        Serial.println("Power on counter >=3, enter binding mode...");
+        EnterBindingMode();
+    }
+
+    // Update the LED while in binding mode
+    if (InBindingMode)
+    {
+        if (millis() > bindLedFlashInterval)
+        {
+            if (LEDPulseCounter == 0)
+            {
+                LED = true;
+            }
+            else if (LEDPulseCounter == 4)
+            {
+                LED = false;
+            }
+            else
+            {
+                LED = !LED;
+            }
+
+            if (LEDPulseCounter < 4)
+            {
+                bindLedFlashInterval = millis() + BIND_LED_FLASH_INTERVAL_SHORT;
+            }
+            else
+            {
+                bindLedFlashInterval = millis() + BIND_LED_FLASH_INTERVAL_LONG;
+                LEDPulseCounter = 0;
+            }
+            
+            
+            #ifdef GPIO_PIN_LED
+            digitalWrite(GPIO_PIN_LED, LED);
+            #endif
+            
+            LEDPulseCounter++;
+        }
+    }
+}
+
+void EnterBindingMode()
+{
+    if ((connectionState == connected) || InBindingMode || webUpdateMode) {
+        // Don't enter binding if:
+        // - we're already connected
+        // - we're already binding
+        // - we're in web update mode
+        Serial.println("Cannot enter binding mode!");
+        return;
+    }
+
+    // Set UID to special binding values
+    UID[0] = BindingUID[0];
+    UID[1] = BindingUID[1];
+    UID[2] = BindingUID[2];
+    UID[3] = BindingUID[3];
+    UID[4] = BindingUID[4];
+    UID[5] = BindingUID[5];
+
+    CRCCaesarCipher = UID[4];
+    DeviceAddr = UID[5] & 0b111111;
+
+    // Start attempting to bind
+    // Lock the RF rate and freq while binding
+    SetRFLinkRate(enumRatetoIndex(RATE_200HZ));
+    Radio.SetFrequency(GetInitialFreq());
+
+    InBindingMode = true;
+
+    Serial.print("Entered binding mode at freq = ");
+    Serial.println(Radio.currFreq);
+}
+
+void ExitBindingMode()
+{
+    if (!InBindingMode) {
+        // Not in binding mode
+        Serial.println("Cannot exit binding mode, not in binding mode!");
+        return;
+    }
+
+    InBindingMode = false;
+
+    // Revert to default packet rate
+    // and go to initial freq
+    SetRFLinkRate(RATE_DEFAULT);
```

Comment:
RFmodeLastCycled = millis();

Try adding this here so that the rf mode isnt instantly incremented in the loop.  We should then pick up a link nearly instantly after binding.